appunite/Loudius
# [LD-161] Add coverage for components module and update excluded dirs.

# Pull-Request
## Description

### Why?

We want to measure test coverage for entire project.

### What?

Added measure coverage for components module and update excluded dirs.

## Links to related issues

- Fixes [LD-161](https://loudius.atlassian.net/browse/LD-161)


## Demo

<img width="924" alt="Screenshot 2024-02-06 at 12 01 07" src="https://github.com/appunite/Loudius/assets/72873966/bcbc421d-ffa7-4975-8bbb-09f78e7d9cf8">

## How to test

Download the code-coverage artifact from GitHub CI and open it. You can see coverage for both - app module and components module. If you want, you can calculate the expected coverage for them and compare with our result, which is 73.13%. (I did it and it's correct ;))

## Documentation
Links to helpers:
- **[Coverage for multi-module app](https://proandroiddev.com/android-code-coverage-on-firebase-test-lab-part-2-advanced-topics-4b64b26e56dc)**
- **[Madrapps](https://github.com/Madrapps/jacoco-report)**
- **[Issue with paparazzi](https://github.com/cashapp/paparazzi/releases/tag/1.3.2)**

## Checklist


~- [ ] Functionality is covered by unit tests~
~- [ ] Functionality is covered by integration tests~
